gr-blocks/python/blocks/qa_hier_block2.py

```diff
@@ -426,5 +427,20 @@ def test_033c_get_affinity(self):
         procs = hblock.processor_affinity()
         self.assertEquals((0,), procs)
 
+    def test_lock_unlock(self):
+        hblock = gr.top_block("test_block")
+        src = blocks.null_source(gr.sizeof_float)
+        throttle = blocks.throttle(gr.sizeof_float, 32000)
+        hier = multiply_const_ff(0.5)
+        sink = blocks.null_sink(gr.sizeof_float)
+        hblock.connect(src, throttle, hier, sink)
+        hblock.set_processor_affinity([0,])
```

Comment:
this is a mistake, wrong cherry-pick, it hould have been test for lock()/unlock() sequence, will push -f fix